main.go

```diff
@@ -123,7 +123,7 @@ func loadDotEnv() {
 	}
 
 	if err := godotenv.Load(); err != nil {
-		log.Fatal("Error loading .env file")
+		log.Fatalf("Error loading .env file: %q", err.Error())
```

Comment:
How does this print? I think just `%s` would be sufficient?

Reply:
Yes.  It prints with quotes around it currently.

Reply:
Can you remove the quotes to make consistent with our other error messages?

Also - do you have an example of an error this would print?

Reply:
In my case it was erroring as such:

```
2019/06/13 19:51:46 Error loading .env file: Can't separate key from value
```